config/ember-try.js

```diff
@@ -1,11 +1,36 @@
 /*jshint node:true*/
 module.exports = {
-  useVersionCompatibility: true,
```

Comment:
I think it's up to you to choose whether to put `useVersionCompatibility` back there or to remove `versionCompatibility` from package.json. Frankly, I totally missed this change during ember init. 😬